tests/s390x_tests/TOOL_s390_chchp/chchpmain.sh

```diff
@@ -0,0 +1,199 @@
+# Copyright (C) 2018 IBM Corp.
+# 
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+
+#!/bin/bash
```

Comment:
It is very uncommone to have a shebang not in the first line _ if this actually works ;)

I also recommend to call `#!/bin/sh -e` to not ignore errors. If you actually need bash and not just POSIX sh then of course `#!/bin/bash -e`

Reply:
Ok, we move shebang to the top :) it works anyway probably because the script is not executed from a console but is being called by other script.

As for "-e" key, I've tried it and it has broken all our script because scripts are written to catch and handle errors when necessary, counting successes and failures. If we add "-e" key, then the script just exits at the first error, even if the error is expected by our its logic (e.g. checks that expects errors to happen, negative checks and so on). That is why better leave it without "-e" key.

Reply:
ok, I see. Thank you for looking into this.